option.go

```diff
@@ -5,14 +5,27 @@ import (
 	"fmt"
 	"reflect"
 	"regexp"
+
+	"emperror.dev/errors"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	defaultSource  = "default"
+	overrideSource = "override"
+	promptSource   = "prompt"
+	yamlSource     = "yaml"
+	jsonSource     = "json"
```

Comment:
Yeah, not sure, that specific question is tricky.  The value will be `yaml` only if the Option was not populated via the figtree config process, but rather populated independenly from yaml.Decode.  If populated from the figree processing it will have `file:line:col` instead. 

I am inclined to leave these private unless someone comes up with a use-case?

Reply:
I'm pretty sure we check for `yaml` at least one place in newt. Are the semantics of when this is set to `yaml changing?

Reply:
No, dont think these symatics have changed, I already updated our usage, we dont look for the literal `yaml` string value.